### ISSUE TITLE
glNext: Add gl::bindBuffer signature analogous to glBindBuffer 

### ISSUE THREAD

**Issue by BanTheRewind** [cinder/Cinder#828]
I'm using TF to update per-instance attribute data like so:

glBindBuffer( GL_TRANSFORM_FEEDBACK_BUFFER, mVboInstanceData );
// TF pass
glBindBuffer( GL_ARRAY_BUFFER, mVboInstanceData );
// Use result as instance attributes

It might be handy to include a signature that simply mirrors the original call, just to have consistent code appearance.

**Comment by richardeakin** [comment]
fwiw, there is [gl::bindBufferBase](https://github.com/cinder/Cinder/blob/glNext/include/cinder/gl/wrapper.h#L218):

```cpp
gl::bindBufferBase( GL_TRANSFORM_FEEDBACK_BUFFER, 0, windVboDest );
gl::ScopedState rasterizerScope( GL_RASTERIZER_DISCARD, true );

gl::beginTransformFeedback( GL_POINTS );
gl::drawArrays( GL_POINTS, 0, numInstances );
gl::endTransformFeedback();
```

I also see [gl::bindBuffer()](https://github.com/cinder/Cinder/blob/glNext/include/cinder/gl/wrapper.h#L301) that takes a BufferObj, perhaps pass that a `gl::TransformFeedbackObj`?

**Comment by BanTheRewind** [comment]
Yeah, I use bindBufferBase for aligning uniform blocks, etc. The problem with passing an object is that it is simply binding the objects data to its target. In my case, I'm taking a generic buffer and changing its target.

This is super low priority. Literally all I'm asking for is this.

```
namespace cinder { namespace gl {
void bindBuffer( GLenum target, GLuint buffer )
{
   glBindBuffer( target, buffer );
}
} }
```

**Comment by andrewfb** [comment]
Thanks for flagging this - fixed in 9e02f1a676049500209e096b8ec038001e62aa57
